### PR TITLE
Fixed Pjax with Listview in CRUD generator

### PR DESCRIPTION
**Pull request by Ariestattoo**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | yes/no
| Tests pass?   | yes/no
| Fixed issues  | #200

…" to the form

### CODE REVIEW

File: generators/crud/default/views/_search.php [hunk 1]
```diff
@@ -22,6 +22,9 @@
     <?= "<?php " ?>$form = ActiveForm::begin([
         'action' => ['index'],
         'method' => 'get',
+    <?php if( $generator->enablePjax){?>'options' => [
```
Comment (SilverFire) [inline generators/crud/default/views/_search.php]: Code formatting should be fixed

` if ($generator->enablePjax) {`

**Comment by Ariestattoo** [comment]
Is there a central code formatting document that I can review for future reference?

**Comment by cebe** [comment]
> Is there a central code formatting document that I can review for future reference?

- https://github.com/yiisoft/yii2/blob/master/docs/internals/core-code-style.md
- https://github.com/yiisoft/yii2/blob/master/docs/internals/view-code-style.md

**Comment by Ariestattoo** [comment]
Is this reconciled?

**Comment by samdark** [comment]
No. Formatting is still wrong.

**Comment by SilverFire** [comment]
Merged with formatting adjustments: a9e6c292e0bf1feb0538280f513e11bc3db876b6

**Pull request closed without merging**